docs/manual/manual_telemetry.rst

```diff
@@ -10,6 +10,12 @@ Flight controllers, including flight controllers running `PX4 <https://px4.io/>`
 about the state of the ESC and the motor(s) it is controlling. Vertiq modules support a range of these telemetry messages across different protocols. This section provides
 details on the structure of the telemetry messages supported by Vertiq modules and how to use them.
 
+.. note:: 
+
+    When using :ref:`redundant throttle sources <redundant_throttle_manual>`, it is still possible to receive module telemetry. Testing has shown that when reporting telemetry through 
+    both DroneCAN and a serial line (:ref:`IQUART Flight Controller Interface <controlling_ifci>`, DSHOT, or PWM), only the telemetry received serially is processed and logged by the 
+    flight controller.

```

Comment:
Is this true for both PX4 and Ardupilot? Just curious. May be worth specifying that if it is true, that those are what the "testing" mentioned here covers and what flight controllers you are referring to, so when someone uses CrazyFlightControllerFirmware and get a different result they know why

Reply:
Gah. I've only noticed it on PX4, I'll check on ArduPilot and update this